RecoLocalTracker/Phase2TrackerRecHits/plugins/Phase2TrackerRecHits.cc

```diff
@@ -77,11 +77,11 @@ void Phase2TrackerRecHits::produce(edm::StreamID sid, edm::Event& event, const e
     // Container for the clusters that will be produced for this modules
     Phase2TrackerRecHit1DCollectionNew::FastFiller rechits(*outputRecHits, DSViter.detId());
 
-    for (auto clustIt : DSViter) { 
-      ClusterParameterEstimator< Phase2TrackerCluster1D >::LocalValues lv = cpe->localParameters(clustIt, *geomDetUnit);
+    for (edmNew::DetSet<Phase2TrackerCluster1D>::const_iterator clustIt = DSViter.begin(); clustIt != DSViter.end(); ++clustIt) { 
```

Comment:
Maybe the range loop should have `const auto&` (i.e. reference instead of copy)?

(same comment for `auto DSViter` on line 70 above)